atom/common/api/event_emitter_caller.cc

```diff
@@ -20,8 +20,13 @@ v8::Local<v8::Value> CallMethodWithArgs(v8::Isolate* isolate,
                                    v8::MicrotasksScope::kRunMicrotasks);
   // Use node::MakeCallback to call the callback, and it will also run pending
   // tasks in Node.js.
-  return node::MakeCallback(isolate, obj, method, args->size(), &args->front(),
-                            {0, 0}).ToLocalChecked();
+  v8::MaybeLocal<v8::Value> ret = node::MakeCallback(isolate, obj, method,
+                                                     args->size(),
+                                                     &args->front(), {0, 0});
+  if (ret.IsEmpty()) {
+    return v8::Boolean::New(isolate, false);
+  }
+  return ret.ToLocalChecked();
```

Comment:
Use of `ToLocalChecked()` is odd here: (1) it involves a redundant `IsEmpty()` test, and (2) its purpose in life is to crash on failure, which we're explicitly avoiding. It makes more sense to use `bool MaybeLocal::ToLocal(Local<S>* out) const` instead.